src/inspector_agent.cc

```diff
@@ -718,6 +874,19 @@ void Agent::InitInspector(Local<Object> target, Local<Value> unused,
   env->SetMethod(target, "connect", ConnectJSBindingsSession);
   env->SetMethod(target, "open", Open);
   env->SetMethod(target, "url", Url);
+
+  env->SetMethod(target, "asyncTaskScheduled", AsyncTaskScheduledWrapper);
+  env->SetMethod(target, "asyncTaskCanceled", AsyncTaskCanceledWrapper);
+  env->SetMethod(target, "asyncTaskStarted", AsyncTaskStartedWrapper);
+  env->SetMethod(target, "asyncTaskFinished", AsyncTaskFinishedWrapper);
+  env->SetMethod(target, "registerAsyncHook", RegisterAsyncHookWrapper);
+
+  auto enabled_string = FIXED_ONE_BYTE_STRING(env->isolate(), "enabled");
+  CHECK(target->SetAccessor(env->context(),
```

Comment:
Sorry, but I believe it is the norm to use a function name like `isEnabled()` for functions exposed into JS. `enabled` sounds like a boolean property, not a function.